### DISCUSSION TITLE
Welcome to LG_Buddy Discussions!

### DISCUSSION THREAD

**Discussion by Faceless3882** [Faceless3882/LG_Buddy#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tazztone** [comment]
hey  

just wondered if we should make a gnome extension that would add another bar there for setting LG TV brightness

<img width="595" height="691" alt="image" src="https://github.com/user-attachments/assets/1e2b7d3d-b460-4858-a82e-29e6a413c3ff" />

**Reply by Staphylococcus** [reply]
I agree with this, in fact I've consolidated the brightness controls internals in the latest pre-release version so things like this could be pulled off more easily and reliably. In 1.0.0, the brightness dialog forges its own patch, but in the pre-release, there is a new CLI interface, with the dialog delegating to the CLI. One could hook into the same dataflow with this potential shell extension, or whatever.

Check this area: https://github.com/Staphylococcus/LG_Buddy/blob/58aca755873682e2043dd112f3ee81f6aa4ec00d/crates/lg-buddy/src/commands.rs#L528

**Reply by Staphylococcus** [reply]
One of the things I'm personally thinking about is: I have brightness slider already present in GNOME. It's not an OLED brightness slider, but a screen brightness slider, but I'm looking at it and thinking that maybe it should be one. I'm not certain one really needs to be able to adjust OLED brightness and screen brightness separately, it's sort of like having 2 dimmers for one light source.
<img width="420" height="426" alt="Screenshot From 2026-05-13 18-18-46" src="https://github.com/user-attachments/assets/07e16ddf-41b1-4a0e-bce4-01d8225b44c8" />

**Comment by Z474AL3MN2VC** [comment]
I tried this a few months ago and it didn't work for me. Today I gave lg-buddy another shot and it works flawlessly! I just wanted to drop a comment to say thank you! Great work team!

LG CX 77, 9070 XT, Fedora 44

**Reply by Staphylococcus** [reply]
There's been some very significant rework in the internal plumbing, great to hear it works better for you now! If you encounter any issues, it would be hugely helpful if you report those.
